tests/AopPostfixClassNameTest.php

```
@@ -0,0 +1,74 @@
+{
+    public function testPostfixStartsWithUnderscore(): void
+    {
+        $className = new AopPostfixClassName(FakeClass::class, 'bindings', '/tmp');
+        $this->assertStringStartsWith('_', $className->postFix);
+    }
+
+    public function testFqnContainsOriginalClassNameAndPostfix(): void
+    {
+        $className = new AopPostfixClassName(FakeClass::class, 'bindings', '/tmp');
+        $this->assertStringStartsWith(FakeClass::class, $className->fqn);
+        $this->assertStringEndsWith($className->postFix, $className->fqn);
```

PHPStan (2.1.32)

Parameter #1 $suffix of method PHPUnit\Framework\Assert::assertStringEndsWith() expects non-empty-string, string given.


(argument.type)